storage/inmem/opts.go

```diff
@@ -0,0 +1,19 @@
+package inmem
+
+// An Opt modifies store at instantiation.
+type Opt func(*store)
+
+// OptRoundTripOnWrite sets whether incoming objects written to store are
+// round-tripped through JSON to ensure they are serializable to JSON.
+//
+// Callers should disable use this if they can guarantee all objects passed to
+// Write() are serializable to JSON. Failing to do so may result in undefined
+// behavior, including panics.
+//
+// If setting to false consider deep-copying any objects passed to Write unless
+// they can guarantee the objects will not be mutated after being written.
```

Comment:
```suggestion
// Callers should disable this if they cannot guarantee all objects passed to
// Write() are serializable to JSON. Failing to do so may result in undefined
// behavior, including panics.
//
// If setting to false, callers should consider deep-copying any objects passed
// to Write, unless they can guarantee the objects will not be mutated after
// being written to the store.
```

Reply:
I've updated the comment in a later commit -- what do you think about that one?

Reply:
One second... I'll go see the latest revisions. :smile:

Reply:
I'm a little confused by the "they" on line 23. I assume it's referring to the caller, and that they have a responsibility to ensure no mutations are happening to objects going into the store?

Reply:
They need to ensure that if the object are mutated (in the store) it won't matter for them. Basically "hand them over"... I'm grateful for all suggestions here 😃

Reply:
I added some extra words in the suggested change block to make it clear it's the caller we're talking about, because when I hear "consider \<action\>" by itself, I usually parse that as a command directed at me, instead of another target.

I know I'm bikeshedding a bit there. :bike: It's probably fine, but clarity is always preferred in comments that may be there for a bit. :smile: